modular_skyrat/code/interactions/lewd/_lewd.dm

```diff
@@ -13,6 +18,11 @@
 #define NORMAL_LUST 10
 #define LOW_LUST 1
 
+#define REQUIRE_NONE 0
+#define REQUIRE_EXPOSED 1
+#define REQUIRE_ANY 2
+#define REQUIRE_UNEXPOSED 3
```

Comment:
If you change these values as such, you lose nothing but gain the option for bit math comparisions in the future if becomes convenient.
```suggestion
#define REQUIRE_NONE 0
#define REQUIRE_EXPOSED 1
#define REQUIRE_UNEXPOSED 2
#define REQUIRE_ANY 3
```